content/docs/common-areas/cdn.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CDN (Content Delivery Network)
+weight: 5
+description: A content delivery network (CDN) is a group of geographically distributed servers that speed up the delivery of web content by bringing it closer to where users are. Data centers across the globe use caching, a process that temporarily stores copies of files, so that you can access internet content from a web-enabled device or browser more quickly through a server near you. CDNs cache content like web pages, images, and video in proxy servers near to your physical location. This allows you to do things like watch a movie, download software, check your bank balance, post on social media, or make purchases, without having to wait for content to load.
+---
+
+# CDN (Content Delivery Network)
```

Comment:
Other things I'd like to see covered here:
- Common CDNs for Drupal
- Domain name considerations

You can take this up later if you want. I'm happy to merge this PR without it.